src/core/sent_packet_metadata.h

```diff
@@ -105,12 +106,21 @@ typedef struct QUIC_SENT_PACKET_METADATA {
     //
     // Frames included in this packet.
     //
-    uint8_t FrameCount;
+    uint8_t FrameCount  : 4;
+    uint8_t PathId      : 4;
```

Comment:
This isn't going to work. PathId is a constantly rotating value. We might have a max of X paths at one time, but paths come and go and this value constantly increases.